src/main/java/htsjdk/tribble/readers/AsciiLineReader.java

```diff
@@ -85,7 +116,10 @@ public int getLineTerminatorLength() {
      * @return A String containing the contents of the line or null if the
      *         end of the stream has been reached
      */
-    public final String readLine(final PositionalBufferedStream stream) throws IOException{
+    // TODO: This should be deprecated and made private/merged with readLine, or made
+    // static, as it reads from the provided input stream rather than the wrapped input stream
+    @Deprecated
```

Comment:
I put an @deprecated inside the javadoc, with the date, and added an @see to the "from" method. I can't figure out how to anchor to the OTHER readLine method though. Maybe I could if I swapped the order of the methods....